sam_bot_description/launch/mapping_launch.py

```diff
@@ -0,0 +1,40 @@
+import os
```

Comment:
I don't see in the tutorial where this file is used / referenced. Can it be removed? If so, then the `mapper_params.yaml` can be as well. If not, use the same `find_shared_package_path` functions to find this file in SLAM Toolbox itself rather than overlaying the yaml file here. 

https://github.com/ros-planning/navigation2/blob/main/nav2_bringup/bringup/launch/navigation_launch.py#L28